src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -3998,6 +4128,48 @@ export class StudyViewPageStore
         );
     }
 
+    private readonly genePanelCounts = remoteData({
+        await: () => [
+            this.selectedSamples,
+            this.molecularProfiles,
+            this.molecularProfileNameSet,
+        ],
+        invoke: async () => {
+            type Response = {
+                [suffix: string]: { value: string; count: number }[];
+            };
+            // This is a partial StudyViewFilter. We any type it because the type declared
```

Comment:
there is a Ts helper which allow us to take a type and make all (or some) of its members optional